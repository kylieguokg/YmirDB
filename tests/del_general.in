set a 1 2 3
set b a 19
list entries
del a
del b
del a
list entries
set x 0 10 20
set y 10 200
append x y
set z 10 20 30 40
push z x
list entries
del x
del y
del z
del x
del y 
list entries
get x
bye
