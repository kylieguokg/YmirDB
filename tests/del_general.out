> ok

> ok

> b [a 19]
a [1 2 3]

> not permitted

> ok

> ok

> no entries

> ok

> ok

> ok

> ok

> ok

> z [x 10 20 30 40]
y [10 200]
x [0 10 20 y]

> not permitted

> not permitted

> ok

> ok

> ok

> no entries

> no such key

> bye
